addons/payment_authorize/views/payment_views.xml

```diff
@@ -9,9 +9,14 @@
                     <group attrs="{'invisible': [('provider', '!=', 'authorize')]}">
                         <field name="authorize_login"/>
                         <field name="authorize_transaction_key" password="True"/>
+                        <field name="authorize_client_key" password="True"/>
+                        <field name="authorize_signature_key" password="True"/>
                         <a colspan="2" href="https://www.odoo.com/documentation/user/online/ecommerce/shopper_experience/authorize.html" target="_blank">How to get paid with Authorize.Net</a>
                     </group>
                 </xpath>
+                <field name="check_validity" position="replace">
```

Comment:
i don't want this, since we can only do it *once* before these inherited views start conflicting

use an onchange on the field that prevents setting this to true + warning:
```python
@api.onchange('provider', 'check_validity')
def onchange_check_validity(self):
  if self.provider == 'authorize':
    self.check_validity = False
    return {'warning': _('This option is not supported for Authorize.net')
```

Reply:
or something like that in spirit